src/style.css

```diff
@@ -1 +1,30 @@
-/* styles go here */
+body {
+  font-size: 16px;
+}
+
+form {
+  margin: 20 auto;
+}
+
+fieldset {
+  margin-bottom: 20px;
+  display: block;
+}
+
+label {
+  margin-bottom: 10px;
+  display: block;
+}
+
+label:last-child {
+  margin-bottom: 0;
+}
```

Comment:
don't use tags to stylize elements, only classes

Reply:
Ok, I changed my approach a bit and rewrote the code.